lighthouse-core/scripts/internal-analysis/analyze-issues.js

```diff
@@ -0,0 +1,275 @@
+
+const HOUR_IN_MS = 60 * 60 * 1000;
+const DAY_FILTER = 90;
+const START_AT = new Date(
+  new Date().getTime() - DAY_FILTER * 24 * HOUR_IN_MS
+).getTime();
+
+const ISSUES_PATH = path.join(__dirname, '../../../.tmp', '_issues.json');
+/** @type {Array<AugmentedGitHubIssue>} */
+const _ISSUES = JSON.parse(fs.readFileSync(ISSUES_PATH, 'utf8')).map(
+  normalizeIssue
+);
+const _ISSUES_SINCE = _ISSUES.filter(
+  issue => new Date(issue.created_at).getTime() > START_AT
+);
+const ISSUES = _ISSUES_SINCE.filter(issue => !issue.pull_request);
+
+/** @param {number} n */
+const percent = n => `${(n * 100).toFixed(1)}%`;
+
+/**
+ * This function only logs *initial* review responses, but could conceivably log all instances of applying "waiting4reviewer".
+ * @param {string} label
+ * @param {Array<AugmentedGitHubIssue>} issues
+ */
+function computeAndLogReviewResponseStats(label, issues) {
+  const initialReviewRequests = issues
+    .map(issue => {
+      const assignEvent = issue.events.find(
+        event => event.event === 'assigned'
+      );
+      const assignee =
+        (assignEvent && assignEvent.assignee && assignEvent.assignee.login) ||
+        '';
+      const firstCommentByAssignee = issue.comments.find(
+        comment => comment.user.login === assignee
+      );
+      const reviewTimeInHours = firstCommentByAssignee
+        ? (new Date(firstCommentByAssignee.created_at).getTime() -
+            new Date(issue.created_at).getTime()) /
+          HOUR_IN_MS
+        : Infinity;
+      return {
+        issue,
+        assignEvent,
+        assignee,
+        firstCommentByAssignee,
+        reviewTimeInHours,
+      };
+    })
+    .filter(review => review.assignee)
+    .sort((a, b) => a.reviewTimeInHours - b.reviewTimeInHours);
+
+  const reviews = initialReviewRequests.filter(
+    review => review.firstCommentByAssignee
+  );
+  const reviewsByLogin = Object.values(
+    initialReviewRequests.reduce(
+      (map, view) => {
+        const reviews = map[view.assignee] || [];
+        reviews.push(view);
+        map[view.assignee] = reviews;
+        return map;
+      },
+      /** @type {Record<string, typeof reviews>} */ ({})
+    )
+  ).sort((a, b) => b.length - a.length);
+
+  const responseTimeInHours = initialReviewRequests.map(r => r.reviewTimeInHours);
+  const medianResponseTime =
+    responseTimeInHours[Math.floor(reviews.length / 2)];
+  console.log(`${log.bold}${label}${log.reset}`);
+  console.log(
+    `  ${percent(
+      initialReviewRequests.length / issues.length
+    )} of PRs requested a review`
+  );
+  console.log(
+    `  ${percent(
+      reviews.length / initialReviewRequests.length
+    )} of requests received a review`
+  );
+  console.log(
+    `  Median initial response time of ${log.bold}${medianResponseTime.toFixed(
+      1
+    )} hours${log.reset}`
+  );
+  console.log('  By User');
+  reviewsByLogin.forEach(requests => {
```

Comment:
the console.table() version of this is

```js
  const byUser = Object.fromEntries(reviewsByLogin.map(requests => {
    const user = requests[0].assignee;
    const reviews = requests.filter(r => r.firstCommentByAssignee);
    const medianResponseTime = requests[Math.floor(reviews.length / 2)].reviewTimeInHours;
    return [user, {reviews: reviews.length, medianResponse: `${medianResponseTime.toFixed(1)} h`}];
  }));
  console.table(byUser);
```

eh. just _slightly_ better IMO

| before | after |
|-|-|
|  ![image](https://user-images.githubusercontent.com/39191/85796081-2d7f2380-b6ee-11ea-8189-7f6d982a963f.png) | ![image](https://user-images.githubusercontent.com/39191/85796109-3839b880-b6ee-11ea-90dc-0d142438a671.png) |